resources/rafter/charts/controller-manager/charts/minio/templates/apirule.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 spec:
-  gateway: {{ .Values.global.istio.gateway.name }}.{{ .Values.global.istio.gateway.namespace }}.svc.cluster.local
```

Comment:
I'm not entirely sure if this resource expects the gateway in this format

Reply:
I checked with @Tomasz-Smelcerz-SAP and indeed it is not supported. He will think about how to make it more convenient.
Anyway, I reverted that change, good finding.